src/openfermion/measurements/_rdm_mapping_functions.py

```diff
@@ -0,0 +1,200 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Mapping RDMs to other RDMs"""
+import numpy
+from itertools import product
+
+
+def krond(i, j):
+    return 1 if i == j else 0
```

Comment:
`float(i == j)` or `int(i == j)` would be cleaner here - I'd slightly prefer the direct float if you don't care otherwise.